docs/source/recipes/index.rst

```diff
@@ -10,8 +10,10 @@ We may add recipes for other tasks as well in the future.
 .. Other recipes are listed in a alphabetical order.
```

Comment:
Could you also change `README.md` at the root directory to add the results for `TIMIT` recipe.

Reply:
Done!  And will finish the colab running after merging.